### ISSUE TITLE
Bug located in ...Cannot enable beta feature in Fluent 23.2

### ISSUE THREAD

**Issue by hauke2007** [pyansys/pyfluent#1584]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Hello I am launching fluent with the following command 

    solver = pyfluent.launch_fluent(precision=turboData["launching"]["precision"], processor_count=int(turboData["launching"]["noCore"]),
                                    mode="solver", show_gui=True,
                                   product_version = turboData["launching"]["fl_version"], cwd=working_Dir)

Fluent comes up and everything looks good but after executing the following command 

solver.tui.define.beta_feature_access("yes yes") I receive the following error message: 

Beta features enabled.
Traceback (most recent call last):
  File "<scheme>", line 1, in <module>
ImportError: cannot import name 'flglobals' from 'setup' (D:\Git\turbotestsuite\setup.py)

Error: Python runtime error!
Error Object: ()
Traceback (most recent call last):
  File "C:\Users\hreese\.ansys_python_venvs\myV310\lib\site-packages\ansys\fluent\core\services\error_handler.py", line 15, in func
    raise RuntimeError(ex.details()) from None
RuntimeError: Python runtime error!
Error Object: ()

Process finished with exit code 1

If I am using the python console inside fluent this does not happen. 

### 📝 Steps to reproduce

execute the following commands: 

import ansys.fluent.core as pyfluent 

solver = pyfluent.launch_fluent(mode="solver", show_gui=True,
                                   product_version = 23.2.0)

solver.tui.define.beta_feature_access("yes ok")

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

Build Time: Apr  4 2023 13:28:44 EDT  Build Id: 176 

### 🐍 Which Python version are you using?

3.10

### 📦 Installed packages

```shell
PS D:\Git\turbotestsuite> python --version
Python 3.10.10
PS D:\Git\turbotestsuite> python -m pip freeze
ansys-api-fluent==0.3.12
ansys-api-platform-instancemanagement==1.0.0b3
ansys-fluent-core==0.14.dev1
ansys-platform-instancemanagement==1.1.1
appdirs==1.4.4
googleapis-common-protos==1.59.0
grpcio==1.54.0
h5py==3.8.0
importlib-metadata==6.6.0
lxml==4.9.2
numpy==1.24.3
pandas==2.0.1
protobuf==3.20.3
protoc-gen-swagger==0.1.0
python-dateutil==2.8.2
pytz==2023.3
PyYAML==6.0
six==1.16.0
tzdata==2023.3
zipp==3.15.0
PS D:\Git\turbotestsuite>
```

**Comment by mkundu1** [comment]
The issue is due to the name conflict between setup.py and a setup module in Fluent Python. The issue will be resolved in Fluent 23.2. Please rename the setup.py or use a different working directory for a temporary workaround.

**Comment by hauke2007** [comment]
Hello Mainak, 
thanks for your fast response and help. With a different file name it works now.
